frontend/isoko/src/components/BusinessPreview.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Rating } from 'react-simple-star-rating';
+import { Table } from 'react-bootstrap';
+import MinorityTag from './MinorityTag';
+import KeywordTag from './KeywordTag';
+
+const BusinessContainer = styled(Table)`
+   width: 85%;
+   height: 25%;
```

Comment:
For a lot of the same reasons as above, I think this should be a px value so that in case this component gets called in a weird place, we will have predictable styles. You might need to mess around with it some, but changing height to like 148px or so might make the responsiveness/predictableness of this component a lot better